Jenkinsfile.treecompose-ootpa

```diff
@@ -105,5 +106,11 @@ node(NODE) {
        stage("Cleanup") { sh """
             rm ${treecompose_workdir} -rf
         """ }
-  }
+    }
```

Comment:
This looks like an extra `}` to me, but I'm not 100% sure. The indentation of groovy throws me off.

Reply:
I put the new `try` blocks at the same indentation level as the rest of the `node` closure to be consistent with the ones that already existed (which I assume did that to not indent everything and have a huge diff).  I indented that one `}` to four spaces because nothing else is indented to two spaces in any of the files.

Reply:
Makes sense. Thanks!